transport/src/main/java/io/netty/channel/SingleThreadEventLoop.java

```diff
@@ -35,7 +35,7 @@
     protected static final int DEFAULT_MAX_PENDING_TASKS = Math.max(16,
             SystemPropertyUtil.getInt("io.netty.eventLoop.maxPendingTasks", Integer.MAX_VALUE));
 
-    private final Queue<Runnable> tailTasks;
+    protected final Queue<Runnable> tailTasks;
```

Comment:
How about adding a protected final getter method with the same name instead of exposing a field?

Reply:
see above comment